OSRMTextInstructions.xcodeproj/project.pbxproj

```diff
@@ -761,7 +761,7 @@
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 1;
```

Comment:
This is the build version. The marketing version is a human-readable string (that isn’t required to conform to a standard like semver), whereas the build version is a machine-readable integer that has to increment with every release. We use `agvtool` to bump the various build settings and Info.plist entries before every release of a library. See [this Technical Q&A](https://developer.apple.com/library/content/qa/qa1827/_index.html) for more details about Apple Generic Versioning.